test/dlc_tests/container_tests/bin/benchmark/run_tensorflow_training_performance_gpu_imagenet

```diff
@@ -15,15 +15,21 @@ mkdir -p ${BIN_DIR}
 mkdir -p ${LOG_DIR}
 
 set -e
-git clone https://github.com/tensorflow/benchmarks.git ${HOME_DIR}/artifacts/tensorflow/benchmarks && cd ${HOME_DIR}/artifacts/tensorflow/benchmarks && git checkout ${GIT_BRANCH}
+git clone https://github.com/tensorflow/benchmarks.git ${HOME_DIR}/artifacts/tensorflow/benchmarks \
+  && cd ${HOME_DIR}/artifacts/tensorflow/benchmarks \
+  && git checkout ${GIT_BRANCH}
 START=$(date +%s)
 TRAINING_TIME="100m"
 if [ "$PR_CONTEXT" == "1" ]
 then
   TRAINING_TIME="20m"
   echo Running under PR context >&2
 fi
-timeout "$TRAINING_TIME" python scripts/tf_cnn_benchmarks/tf_cnn_benchmarks.py --batch_size=256 --model=resnet50_v1.5 --optimizer=momentum --variable_update=replicated --nodistortions --gradient_repacking=2 --num_gpus=8 --num_epochs=91 --weight_decay=1e-4 --use_fp16 --all_reduce_spec=nccl --save_summaries_steps=0 --summary_verbosity=1 --num_warmup_batches=0 --train_dir=$HOME/test00 --compute_lr_on_cpu=True --single_l2_loss_op=True --loss_type_to_report=base_loss --data_name=imagenet 2>&1 | tee "${LOG_FILE}"
+timeout "$TRAINING_TIME" python scripts/tf_cnn_benchmarks/tf_cnn_benchmarks.py --batch_size=256 --model=resnet50_v1.5 \
+        --optimizer=momentum --variable_update=replicated --nodistortions --gradient_repacking=2 --num_gpus=8 \
+        --num_epochs=8 --weight_decay=1e-4 --use_fp16 --all_reduce_spec=nccl --save_summaries_steps=0 \
```

Comment:
Changed `--num-epochs` to 8 from 91.

Reply:
The num of epoch is changed so that the throughput becomes much lower